modules/transactionpool/transactionpool.go

```diff
@@ -102,9 +104,12 @@ func New(cs modules.ConsensusSet, g modules.Gateway, persistDir string) (*Transa
 	if err != nil {
 		return nil, err
 	}
+	if build.DEBUG && build.Release != "testing" {
+		fmt.Println("\tTransaction Pool: persist initialization complete")
+	}
 
 	// Register RPCs
-	g.RegisterRPC("RelayTransactionSet", tp.relayTransactionSet)
+	go g.RegisterRPC("RelayTransactionSet", tp.relayTransactionSet)
```

Comment:
did it help?
my concern is that you have a race condition if you try to call the `RelayTransactionSet` RPC immediately after creating the transaction pool, though I'm sure this would only occur during testing and in race circumstances.